run_beir.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+# List of datasets
+datasets=("beir_cqadupstack-gis" "beir_cqadupstack-mathematica" "beir_cqadupstack-physics" "beir_cqadupstack-programmers" "beir_cqadupstack-stats" "beir_cqadupstack-tex" "beir_cqadupstack-unix" "beir_cqadupstack-webmasters" "beir_cqadupstack-wordpress" "beir_dbpedia-entity" "beir_fever" "beir_fiqa" "beir_hotpotqa" "beir_nfcorpus" "beir_nq" "beir_quora" "beir_robust04" "beir_scidocs" "beir_scifact" "beir_signal1m" "beir_trec-covid" "beir_trec-news" "beir_webis-touche2020")
+
+# List of GPUs
+GPUS=(0 1)
+
+# Function to run the command on a specific GPU
+run_on_gpu() {
```

Comment:
This relies on pyserini retriever, right? @lintool wants this retriever to be removed. Unless we decide to keep it, we should not add more dependencies to it.

Reply:
Oops old stuff, removed